docs/sdk/serverless-signing/encryption.md

```diff
@@ -0,0 +1,53 @@
+import FeedbackComponent from "@site/src/pages/feedback.md";
+
+# Decrypting and Combining Within an Action
+
+## Overview
+
+Within a Lit action, you may choose to `encrypt` from within the context of an action. Encryption within the action may be useful if you do not wish to provide context on the encryption material to the client. Keeping all context on the encryption operation within the Trusted Execution Enviorment (TEE).
+
+::: note
+The lit action implementation of `encrypt` will not return the `dataToEncryptHash` this can be created with the `crypto.subtle.digest` implementation by specifying the `SHA-256` hashing option if your use case requires the message hash.
```

Comment:
```suggestion
The Lit Action implementation of `encrypt` will not return the `dataToEncryptHash`. This can be created with the `crypto.subtle.digest` implementation by specifying the `SHA-256` hashing option if your use case requires the message hash.
```